src/gen/direct_wrappers/gsl_integration_h.jl

```diff
@@ -1041,7 +1041,7 @@ GSL documentation:
 
 ### `void gsl_integration_fixed_free(gsl_integration_fixed_workspace * w)`
 
-> This function frees the memory assocated with the workspace `w`
+> This function frees the memory associated with the workspace `w`
```

Comment:
Everything in this subdirectory is generated from the GSL docs, so we should propose those changes there as well (otherwise, if we upgrade the bindings we will lose those changes).

Reply:
I sent an email to bug-gsl@gnu.org, hopefully they can fix it.

Reply:
> this subdirectory

src/gen/direct_wrappers/*.jl